x-pack/plugins/cross_cluster_replication/public/app/components/follower_index_pause_provider.js

```diff
@@ -56,11 +56,11 @@ class Provider extends PureComponent {
     const title = isSingle
       ? intl.formatMessage({
         id: 'xpack.crossClusterReplication.pauseFollowerIndex.confirmModal.pauseSingleTitle',
-        defaultMessage: 'Pause follower index \'{name}\'?',
+        defaultMessage: 'Pause replication of follower index \'{name}\'?',
```

Comment:
this sounds like we are pausing replication of the follower index itself, rather than saying that the follower index will stop replicating its leader index. what do you think of saying something like `Pause replication tasks of...` here and elsewhere in this file? cc @gchaps

Reply:
@jen-huang, @cjcenizal   I like the idea of adding the word task.  I tried it out:

**Modals**

Pause replication tasks of follower index 'd'?

Pausing the replication tasks of this follower index clears its custom advanced settings.

Resume replication tasks of follower index "d"?

Replication tasks resume using the default advanced settings...

**Menu**

Pause/Resume replication tasks

Reply:
@cjcenizal I thought it was "replication _to_"? 🤔  like `Pause replication to follower index...`

Reply:
Bah, typo on my part. Thanks for catching that! Edited the comment.

Reply:
"replication to" feels cleaner.  I was worried about the modal titles getting too long.